WeeklyTask/WeeklyTask/CreateTask.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WeeklyTask
+{
+    class CreateTask
```

Comment:
class name should not contin verbs, use `CommonTask`, `BaseTask`, `RegularTask` etc